logger.c

```diff
@@ -371,6 +371,59 @@ static int _logger_parse_prx_req(logentry *e, char *scratch) {
             );
     return total;
 }
+
+static void _logger_log_proxy_errbe(logentry *e, const entry_details *d, const void *entry, va_list ap) {
+    char *errmsg = va_arg(ap, char *);
+    char *be_name = va_arg(ap, char *);
+    char *be_port = va_arg(ap, char *);
+    char *be_rbuf = va_arg(ap, char *);
+    int be_rbuflen = va_arg(ap, int);
+
+    struct logentry_proxy_errbe *le = (void *)e->data;
+    le->errlen = strlen(errmsg);
+    if (be_name && be_port) {
+        le->be_namelen = strlen(be_name);
+        le->be_portlen = strlen(be_port);
+    }
+
+    le->be_rbuflen = be_rbuflen;
+    if (be_rbuflen > 100) {
+        le->be_rbuflen = 100;
```

Comment:
Is this `100` value related to MAX_RBUF_READ?

Reply:
No I was just putting random limits down. I'll make this more consistent today. 100 is longer than any error message would be, is why I picked that.

Reply:
It is semi-related though; it's 300 / 3 (worst case URI encoding)

Reply:
Thanks, that makes sense. Just from a stand point of getting rid of magic number in the code.

Reply:
moved the define around